package.json

```
@@ -40,23 +40,23 @@
   "dependencies": {
     "@emotion/react": "^11.1.5",
     "@emotion/styled": "^11.1.5",
-    "@fortawesome/fontawesome-free": "^5.15.2",
-    "@fortawesome/fontawesome-svg-core": "^1.2.34",
-    "@fortawesome/free-solid-svg-icons": "^5.15.2",
+    "@fortawesome/fontawesome-free": "^5.15.3",
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
     "@fortawesome/react-fontawesome": "^0.1.14",
-    "@hookform/resolvers": "1.3.0",
+    "@hookform/resolvers": "1.3.5",
```

Maybe we could not update `@hookform/resolvers` in order to don't break our tests.
```suggestion
    "@hookform/resolvers": "1.3.0",
```

---

@nicocadq Correct, we've been doing this waiting for the `2.0.0` release which is getting released soon: https://github.com/react-hook-form/resolvers/releases